skycoin/skywire
# Improvements for the Skychat UI

Did you run `make format && make check`?
No go code was changed.

Fixes #599 

 Changes:	
- The app now saves the chat history.
- Now it is possible to add profile images to the chats.
- Now the UI shows the messages as bubbles.
- Now it is not possible to send messagess if no chat has been selected.
- Now the apps show a basic count of unreaded messages.
- Problems with the scroll areas were fixed.
- Now the app does not show received messages outside the appropriate chat areas.
- Several small bugs were solved.
- Several additional small usability improvements.

How to test this PR:
Open the new Skychat UI and check it.

NOTE: the new UI does not appear after compilation just by having the new files in `cmd/apps/skychat/static`. This appears to be because the files must be embedded in a go file by an additional step.
